content/applications/hr/appraisals/skills_evolution.rst

```diff
@@ -0,0 +1,149 @@
+================
+Skills evolution
+================
+
+In Odoo's **Appraisals** app, it is possible to view employee's skills as they progress over time in
+the :ref:`Skills Evolution <appraisals/identify-skill-evolution>` report.
+
+This is helpful to managers, so they can see who is are achieving their various skill goals set on
+their appraisals, who is meeting their skill deadlines, who has the highest performance in terms of
+skill development, and more.
+
+The *Skills Evolution* report also allows for :ref:`searching employees with specific skills
+<appraisals/identify-skills>` at certain levels, which can be helpful for scenarios where specific
+skills are required.
+
+.. _appraisals/identify-skill-evolution:
+
+Skills evolution report
+=======================
+
+To access this *Skills Evolution* report, navigate to :menuselection:`Appraisals app --> Reporting
+--> Skills Evolution`.
+
+The :guilabel:`Appraisal Skills Report` page displays a report of all skills, grouped by employee,
+in alphabetical order, by default.
+
+.. note::
+   Skill levels are **only** updated after an appraisal is marked as done. Any skill level changes
+   from any ongoing appraisals that have **not** been finalized are **not** included in this report.
+
+All the :guilabel:`Employee` lines are expanded with all the various skill types nested below. Each
+individual skill type is collapsed, by default. To view the individual skills contained within a
+skill type, click anywhere on the skill type line to expand the data.
+
+Each skill has the following information listed:
+
+- :guilabel:`Employee`: The name of the employee.
+- :guilabel:`Skill Type`: The category the skill falls under.
+- :guilabel:`Skill`: The specific, individual skill.
+- :guilabel:`Previous Skill Level`: The level the employee had previously achieved for the skill.
+- :guilabel:`Previous Skill Progress`: The previous percentage of competency achieved for the skill
+  (based on the :guilabel:`Skill Level`).
+- :guilabel:`Current Skill Level`: The current level the employee has achieved for the skill.
+- :guilabel:`Current Skill Progress`: The current percentage of competency achieved for the skill.
+- :guilabel:`Justification`: Any notes entered on the skill, explaining the progress.
+
+The color of the skill text indicates any changes from the previous appraisal. Skill levels that
+have increased since the last appraisal appear in green as an *Improvement*, skill levels that have
+**not** changed appear in black as *No Change*, and skills that have regressed appear in red as
+*Regression*.
```

Comment:
```suggestion
have increased since the last appraisal appear in green, as an *Improvement*. Skill levels that have
**not** changed appear in black, as *No Change*. Skills that have regressed appear in red, as
*Regression*.
```